### DISCUSSION TITLE
Allow certain roles to access certain page and links

### DISCUSSION THREAD

**Discussion by joeytzy** [Budibase/budibase#11514] [Ideas]
For the navigation link, I noticed that each link can set a minimum role such as basic.
![image](https://github.com/Budibase/budibase/assets/44018390/4ec732e0-1421-4f17-a857-5a19580e1998)

Would it be better to allow certain roles to access it instead of checking on the minimum role. For example, power user should not be able to see men and women links and should only be able to see the manage. So instead of using minimum role to restrict pages, we can specify which roles can access and which role cant

**Comment by ConorWebb96** [comment]
Hi @joeytzy,

There were a lot of changing to the RBAC (role based access control) recently. I don't think a change like this is likely to happen in the foreseeable future. But feel free to create a feature request here; https://github.com/Budibase/budibase/issues

Thanks for your input, I'll forward this onto our product team!

**Reply by melohagan** [reply]
There is also an open related issue here: https://github.com/Budibase/budibase/issues/10645
